ansible/roles/keycloak-provision/templates/keycloak-service.j2

```diff
@@ -83,7 +83,7 @@ function start_keycloak() {
         # Make sure log files exist and are writable by $PDIUSER first
         touch $LOGOUT $LOGERR
         chown $keycloakuser:$keycloakuser $LOGOUT $LOGERR
-        su - $keycloakuser -c "cd $keycloakpath && (nohup sh ./standalone.sh -b 0.0.0.0 -bprivate=$(hostname -i) --server-config standalone-ha.xml 0<&- 1>>$LOGOUT 2>>$LOGERR &)"
+        su - $keycloakuser -c "cd $keycloakpath && (nohup sh ./standalone.sh -b={{ansible_default_ipv4.address}} -bprivate={{ansible_default_ipv4.address}} --server-config standalone-ha.xml 0<&- 1>>$LOGOUT 2>>$LOGERR &)"
```

Comment:
if keycloak behind an LB, it'll fail. Because keycloak want requests for keycloak IP, but the requests are coming to lb IP. Did you check the scenario? I think it's safe to use 0.0.0.0 as this is a private server

Reply:
Yes it works behind an LB also. We have tested this in loadtest env. Moreover, in clustering mode, especially with JDBC Ping, we should not run on 0.0.0.0. It will cause occasional issues in cluster communications.

Reply:
cool.